Rakefile

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env rake
+
+chef_version = '12.19.36'
+namespace :style do
+  require 'rubocop/rake_task'
+  desc 'Style check with rubocop'
+
+  require 'foodcritic'
+  desc 'Style check with foodcritic for root'
+  orig_dir = Dir.pwd
+  FoodCritic::Rake::LintTask.new('chef./'.to_sym) do |t|
```

Comment:
Rake tasks around the chefdk ecosystem tends to be less maintained than the main CLI scripts.

Problems in the broader community tent to occur in some versions of ChefDK where each individual tool gets rubygem activation conflicts.

CLI applications work around this by using appbundler where each CLI app can activate its own set of dependencies.  (At least that's how I understood how chefdk works)

TLDR it's probably better to shell out

Reply:
I have updated to shell out now.